cli/src/main/java/org/jboss/pnc/build/finder/cli/Main.java

```diff
@@ -437,7 +437,6 @@ private void initCaches(BuildConfig config) {
         GlobalConfiguration globalConfiguration = globalConfig.build();
         Configuration configuration = new ConfigurationBuilder().expiration()
                 .lifespan(config.getCacheLifespan())
-                .maxIdle(config.getCacheMaxIdle())
```

Comment:
This removes an unrelated warning from the log, but probably doesn't belong here.